openapi-cli/src/main/java/io/ballerina/openapi/cmd/Sanitize.java

```diff
@@ -54,7 +54,7 @@ public String getDefaultFileName() {
     @Override
     public Optional<OpenAPI> generate(String openAPIFileContent) {
         Optional<OpenAPI> filteredOpenAPI = getFilteredOpenAPI(openAPIFileContent);
-        return filteredOpenAPI.flatMap(this::getFlattenOpenAPI).flatMap(this::sanitizeOpenAPI);
+        return filteredOpenAPI.flatMap(this::sanitizeOpenAPI);
```

Comment:
Need to update the command description as well: https://github.com/ballerina-platform/openapi-tools/blob/3434ba64067c064f8263408b9b1c039dff90c6a0/openapi-cli/src/main/resources/cli-help/ballerina-openapi-sanitize.help#L15